lib/cog/adapters/slack/rtm_connector.ex

```diff
@@ -108,7 +108,10 @@ defmodule Cog.Adapters.Slack.RTMConnector do
                               fn() -> handle_command(room, user_id, text, state) end}
                          end
     unless message == nil do
-      Slack.API.send_message(channel, message)
+      {:ok, room} = Slack.API.lookup_room(id: channel)
```

Comment:
It looks like `Slack.API.send_message/2` only requires an `id` key in the map, so you could skip the lookup and just send in `%{"id" => channel}` as `room`.